### PR TITLE
Display teams

### PR DESCRIPTION
**Pull request by alemannosara**
Displays all the groups the user is into and allows the user to add a new team with name and list of users.

### CODE REVIEW

**Comment by alemannosara** [comment]
This is not yet completed as there is a bug when checking how much users are added. Also, the description is not completed as the "fromId" method does not yet exist on master.

**Comment by alemannosara** [comment]
@KnightEther do you think that the fact the list is static can influence the behavior of the app when we unselect a user? 

I am trying to fix the following problem : 
when we click on two users to add them to the team and we unselect one of them, the list is not updated.

**Comment by KnightEther** [comment]
> @KnightEther do you think that the fact the list is static can influence the behavior of the app when we unselect a user?
> 
> I am trying to fix the following problem :
> when we click on two users to add them to the team and we unselect one of them, the list is not updated.

I did not check what you've change for know but at the creation of the selector, the list clear so it should not be a problem (maybe we should clear it on destroy as well)

File: app/src/main/java/ch/epfl/sweng/thegramginers/Teams/TeamList/TeamListAdapter.java [hunk 1]
```diff
@@ -26,13 +26,13 @@ public TeamListAdapter(List<Team> teams) {
...
         LayoutInflater inflater = LayoutInflater.from(context);
-        View view = inflater.inflate(R.layout.layout_group_description, viewGroup, false);
+        View view = inflater.inflate(R.layout.layout_team_description, viewGroup, false);
         return new TeamListViewHolder(view);
     }
 
     @Override
     public void onBindViewHolder(@NonNull TeamListViewHolder groupListViewHolder, int i) {
-        groupListViewHolder.updateWithGroup(this.teams.get(i));
+        groupListViewHolder.updateWithTeam(this.teams.get(i));
```
Comment (KnightEther) [inline app/src/main/java/ch/epfl/sweng/thegramginers/Teams/TeamList/TeamListAdapter.java]: Are we sure that the team is in the cache? if not we may want to wait that it is synchronised before updating

**Comment by alemannosara** [comment]
Still needs to be done : 
- do so that the user doesn't see hisself when selecting the user of a new group
- fix the Long to Integer conversion

**Pull request merged**